### DISCUSSION TITLE
Not able to apply for the GitHub Student Education pack.

### DISCUSSION THREAD

**Discussion by joden0503** [community/community#131706] [GitHub Education]
I recently made an account on GitHub for my Harvard CS50 course and came across the GitHub Student Education pack. I tried to apply for it both through my laptop and mobile but it's facing technical issues. On my laptop, I'm not able to upload my student ID as the page is not loading and on mobile it's showing something went wrong try again.![IMG-20240710-WA0013.jpg](https://github.com/community/community/assets/175145272/e143d2c7-fc46-4175-8e45-f3c44a97ff97)

![Screenshot_2024-07-10-21-19-28-10_320a9a695de7cdce83ed5281148d6f19.jpg](https://github.com/community/community/assets/175145272/57780bc8-4dc2-4318-aae3-6247de1a6c8e)

**Comment by Mr-sunandmoon** [comment]
目前找不到太好的方法
